src/components/AddButton.jsx

```diff
@@ -0,0 +1,161 @@
+import ListItemIcon from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import { extend as extendI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import AppLinker, { generateWebLink } from 'cozy-ui/transpiled/react/AppLinker'
+
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+
+const FLAG_FAB_ACTIONS = 'home.fab.actions'
+
+const filterAvailableActions = actions => {
+  return actions.filter(action =>
+    action.flag ? flag(action.flag.name).toString() === action.flag.value : true
+  )
+}
+
+const reworkActions = (actions, apps) => {
+  if (apps.length === 0) return []
+
+  const storeApp = apps.find(app => app.slug === 'store')
+
+  return actions.map((action, index) => {
+    if (action.divider) {
+      return {
+        ...action,
+        id: index
+      }
+    } else {
+      const actionApp = apps.find(app => app.slug === action.slug)
+      const actionId = `${action.slug}${action.path}`
+
+      if (actionApp) {
+        return {
+          ...action,
+          id: actionId,
+          app: actionApp
+        }
+      } else if (storeApp) {
+        return {
+          ...action,
+          id: actionId,
+          app: storeApp,
+          slug: storeApp.slug,
+          path: `/discover/${action.slug}`
+        }
+      }
+    }
+  })
+}
+
+const extendDict = (actions, lang) => {
+  actions.forEach(action => {
+    if (!action.divider) {
+      extendI18n({
+        [action.id]: action.text[lang] || action.text['en']
+      })
+    }
+  })
+}
+
+export const AddButton = () => {
+  const { t, lang } = useI18n()
+
+  const anchorRef = React.createRef()
+
+  const { data: installedAppsData } = useQuery(appsConn.query, appsConn)
+
+  const installedApps = installedAppsData || []
+
+  const [isOpen, setIsOpen] = useState(false)
+  const showMenu = () => setIsOpen(true)
+  const hideMenu = () => setIsOpen(false)
+
+  const flagActions = flag(FLAG_FAB_ACTIONS) || []
+  const availableActions = filterAvailableActions(flagActions)
+  const reworkedActions = reworkActions(availableActions, installedApps)
+
+  extendDict(reworkedActions, lang)
+
+  const client = useClient()
+
+  const { origin: cozyUrl } = new URL(client.getStackClient().uri)
+  const { subdomain: subDomainType } = client.getInstanceOptions()
+
+  return (
+    reworkedActions.length > 0 && (
+      <div className="add-button">
+        <Fab
+          color="primary"
+          aria-label="add"
+          onClick={showMenu}
+          ref={anchorRef}
+        >
+          <Icon icon="plus" size={16} />
+        </Fab>
+        {isOpen && (
+          <BottomSheet anchorElRef={anchorRef} backdrop onClose={hideMenu}>
+            <BottomSheetHeader className="u-flex-justify-center u-pb-1">
+              <Typography variant="h6">Ajouter</Typography>
```

Comment:
idem ici on pourrait avoir un BottomSheetTitle par exemple

Reply:
Quand on dit "on pourrait avoir un BottomSheetTitle par exemple", ce serait de rajouter ça dans cozy-ui ?

Reply:
Oui rajouter un BottomSheetTitle dans cozy-ui, qui prendrai une prop label par exemple, et qui ne ferait qu'un `              <Typography variant="h6">{label}</Typography>`

Reply:
Après discussion, créer un BottomSheetTitle centré qui se place dans un BottomSheetItem

Reply:
@Crash--  We are now just waiting this BottomSheetTitle https://github.com/cozy/cozy-ui/pull/2309. PR is still in draft because I need to talk with @JF-Cozy about cozy-ui implementation and good practices.

But the button can also be merged as is.